python/tests/unit/arcticdb/version_store/test_update.py

```diff
@@ -468,3 +469,76 @@ def test_update_pickled_data(lmdb_version_store):
     df2 = pd.DataFrame({"a": [1000]}, index=idx[1:2])
     with pytest.raises(InternalException) as e_info:
         lmdb_version_store.update(symbol, df2)
+
+
+def test_update_non_sorted_exception(lmdb_version_store):
```

Comment:
I think we also need a test that writes some `UNSORTED` data, and then calls `update` with some sorted data, as this should fail.

Reply:
Initially we were doing it, but after some discussions with Matt we moved towards explicitly check only the new data, not the existing one. I will leave this comment open, so we remember to discuss it next week

Reply:
`update` should fail regardless of whether the data is sorted or not. Agree this would be a good descriptive test, but given the contents of the data doesn't matter this needn't block the PR IMO.

Reply:
Spoke with @GemDot. Code doesn't currently reject updates if the existing data is unsorted. Modifying it to do that and adding a test.